### DISCUSSION TITLE
Not reopens previous file or folder

### DISCUSSION THREAD

**Discussion by tujeff** [gitpod-io/openvscode-server#279] [Q&A]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.63.2
- OS Version: Debian Bullseye

Steps to Reproduce:

1. Open a folder, close the browser
2. Reopen the openvscode server URL

Openvscode never seems to remember my previously opened folder and does always start with an empty or startup window.

**Comment by jeanp413** [comment]
@tujeff folder is passed as a query parameter in the url e.g. `http://localhost:9888/?folder=vscode-remote%3A%2F%2Flocalhost%3A9888%2Fhome%2Fjeanpierre%2Fgitrepos%2Fdeno` so if you open the same url with the same query the folder should be opened from the start

**Comment by tujeff** [comment]
@jeanp413 Understood! It means the user has to save the whole URL include query as a bookmark, which is kind of strange. Is that a way openvscode can save or remember our last opened folder internally? So the user can just enter the simple URL e.g. https://openvscode.mysite.com
